lib/logstash/inputs/twitter.rb

```diff
@@ -86,34 +86,13 @@ def run(queue)
     @logger.info("Starting twitter tracking", :keywords => @keywords)
     begin
       @client.filter(:track => @keywords.join(",")) do |tweet|
```

Comment:
At my understanding, this method is basically doing a post every few times, isn't? https://dev.twitter.com/streaming/reference/post/statuses/filter so I understand this is not any kind of blocking call, right?